pkg/tsdb/cloudwatch/log_sync_query.go

```diff
@@ -107,8 +113,8 @@ func (e *cloudWatchExecutor) syncQuery(ctx context.Context, logsClient cloudwatc
 		if isTerminated(*res.Status) {
 			return res, err
 		}
-		if attemptCount >= alertMaxAttempts {
-			return res, fmt.Errorf("fetching of query results exceeded max number of attempts")
+		if (time.Duration(attemptCount) * time.Second) >= time.Duration(logsTimeout) {
```

Comment:
What is this part? `(time.Duration(attemptCount) * time.Second)`
what do you want to compare? I think this might be a typo

Reply:
Since the attempts happen every second, Im just converting attemptCount to time.Duration so I can multiply it with second. I guess I dont need the parentheses so `time.Duration(attemptCount) * time.Second >= time.Duration(logsTimeout)` is better

Reply:
discussed with Ida, thanks for sharing this common gotcha with me https://stackoverflow.com/questions/17573190/how-to-multiply-duration-by-integer